src/main/java/io/vlingo/schemata/model/SchemaVersion.java

```diff
@@ -77,6 +78,7 @@ public String toString() {
       public boolean isRemoved() { return true; }
     };
 
+    public final String value = this.name();
```

Comment:
I had to add this because I don't know how to make it work with `SqlStatement::bindFields`